game/physics/collisionworld.cpp

```diff
@@ -336,6 +336,10 @@ void CollisionWorld::setBBox(const btVector3& min, const btVector3& max) {
   bbox[1] = max;
   }
 
+float CollisionWorld::worldHeight() const {
```

Comment:
this function doesn't do, what it claims. In case, if `bbox[1].y` is zero - all physics will break